packages/webapp/src/pages/play.tsx

```diff
@@ -32,30 +42,34 @@ const Play: NextPage = () => {
     setLoading,
     // Optimistically update the UX, as we know the transaction succeeded and the data refresh
     // will follow.
-    onSuccess: () => setConcedeCompleted(true)
-  })
+    onSuccess: () => setConcedeCompleted(true),
+  });
 
-  const ctrl = useModalController({ displayed: true, closeable: false })
+  const ctrl = useModalController({ displayed: true, closeable: false });
 
   // TODO: if there is no game ID, should redirect away from this page
   // TODO: the navbar connector should show bad chain if it's a bad chain
 
   // -----------------------------------------------------------------------------------------------
 
-  if (!hydrated) return <></>
+  if (!hydrated) return <></>;
 
   return (
     <>
-      {loading && <LoadingModal ctrl={ctrl} loading={loading} setLoading={setLoading} />}
+      {loading && (
+        <LoadingModal ctrl={ctrl} loading={loading} setLoading={setLoading} />
+      )}
 
-      {ended && !hideResults && <GameEndedModal closeCallback={() => setHideResults(true)} />}
+      {ended && !hideResults && (
+        <GameEndedModal closeCallback={() => setHideResults(true)} />
+      )}
 
       <main className="flex min-h-screen flex-col">
         <Navbar />
 
         <Hand
           cards={playerHand}
-          className="mt-500 absolute z-[100] translate-y-1/2 transition-all duration-500 ease-in-out hover:translate-y-0"
+          className="absolute left-0 right-0 mx-auto z-[100] translate-y-1/2 transition-all duration-500 rounded-xl ease-in-out hover:translate-y-0"
```

Comment:
Also getting `PrivateInfo` from store, populating `PlayerHand` which is fed into `Hand`.

Reply:
Oh I see! I think for me it either errored or didn't show any card because I had to set the `cards` manually. Which can be fine for testing, instead of pulling the deck in.